calico-cloud_versioned_docs/version-3.17/releases.json

```diff
@@ -1,259 +1,263 @@
 [
   {
-    "title": "master",
+    "title": "v3.17.1",
     "tigera-operator": {
       "image": "tigera/operator",
-      "version": "master",
+      "version": "v1.30.5",
       "registry": "quay.io"
     },
     "calico": {
-      "minor_version": "v3.24",
+      "minor_version": "v3.25",
```

Comment:
should be 3.26

Reply:
Fixed.

Reply:
@rene-dekker reverted back to 3.25 per discussion.

Reply:
This should at least be v3.26. The operator that installs v3.17.1 installs Calico OS v3.26.1 components.

Reply:
To clarify why v3.25, because the Enterprise images were based on Calico v3.25 images, not v3.26.